lomap/dbmol.py

```diff
@@ -120,90 +120,65 @@ def __init__(self, directory, parallel=1, verbose='off',
         max_dist_from_actives : int
             The maximum number of links from any molecule to an active
 
-
         """
-
         # Set the Logging
         if verbose == 'off':
             logging.basicConfig(format='%(message)s', level=logging.CRITICAL)
-
-        if verbose == 'info':
+        elif verbose == 'info':
             logging.basicConfig(format='%(message)s', level=logging.INFO)
-
-        if verbose == 'pedantic':
+        elif verbose == 'pedantic':
             logging.basicConfig(format='%(message)s', level=logging.DEBUG)
             # logging.basicConfig(format='%(levelname)s:\t%(message)s', level=logging.DEBUG)
 
-        if __name__ == '__main__':
-            self.options = parser.parse_args()
```

Comment:
@JenkeScheen this is the only breaking change.  If you're using this I can probably put it back in, but I thought it was a little ugly.  The other change is that `.options` is a dict not a namespace, so access is a little different, but items have the same name.  If that breaks a lot of things for you I could hack together a quasi dict that allows `.attribute` access

Reply:
I'm currently using the cresset fork of LOMAP that lives in [BioSimSpace/Align](https://github.com/michellab/BioSimSpace/tree/devel/python/BioSimSpace/Align/_lomap) so my current work isn't affected by changes made in the Mobley repo (most of my work has been in a function that pulls data from LOMAP, not changing it, see [BSS.Align.generateNetwork()](https://github.com/michellab/BioSimSpace/blob/devel/python/BioSimSpace/Align/_align.py#L82))